tests/fast-integration/telemetry-test/telemetry-test.js

```diff
@@ -169,17 +169,6 @@ describe('Telemetry Operator tests, prepare the environment', function() {
     assert.isTrue(logsPresent, 'No logs received by mockserver present in Loki');
   });
 
-  it('Include kyma-system namespace on loki pipeline ', async () => {
```

Comment:
this test is not necessary anymore with https://github.com/kyma-project/kyma/pull/15030